test/edu-benefits/1990-rjsf/config/servicePeriods.unit.spec.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1990-rjsf/config/form';
+
+describe('Edu 1995 servicePeriods', () => {
+  const { schema, uiSchema } = formConfig.chapters.militaryHistory.pages.servicePeriods;
+  const definitions = formConfig.defaultDefinitions;
+  it('should render', () => {
```

Comment:
This test looks identical to the next one, just without an exact count of fields. Maybe we should remove it?